src/platforms/apple/common/enriching-events/user-feedback.mdx

```diff
@@ -31,3 +31,30 @@ userFeedback.email = @"john.doe@example.com";
 userFeedback.name = @"John Doe";
 [SentrySDK captureUserFeedback:userFeedback];
 ```
+
+To capture user feedback of a crash you can use the callback `onCrashedLastRun` of the `SentryOptions`.
```

Comment:
```suggestion
For capturing the User's feedback (only for crashes), you can use the `SentryOptions.onCrashedLastRun` callback.
```

Reply:
not sure if it's better, but it reads better on my mind.

Reply:
I would rather stick to the existing version. Maybe @PeloWriter you can help us here a bit with the wording?